### ISSUE TITLE
Error with drawProjection

### ISSUE THREAD

**Issue by Mahmouded** [jintonic/gears#15]
Hi,
I'm getting an error when I try to use /score/drawProjection to visualize a boxMesh. I'm using the VRML vis. The error is as the following:

```
/vis/enable
/score/drawProjection boxMesh dDep

-------- WWWW ------- G4Exception-START -------- WWWW -------
*** G4Exception : VRML-2001
      issued by : G4VRML2SCENEHANDLER::AddPrimitive (const G4Polyline&)
2D polylines not implemented.  Ignored.
*** This is just a warning message. ***
-------- WWWW -------- G4Exception-END --------- WWWW -------
```

Is this feature available only with OpenGL?

Thanks,

**Comment by jintonic** [comment]
I compiled `Geant4/11.1.0/share/Geant4/examples/extended/runAndEvent/RE03` and changed the vis driver in run1.mac to `VRML2FILE` and saw the same warning message. I opened the resulting `g4_00.wrl` and could not find anything related to the mesh. When I switch back to OGL or TSG, the message disappears and I can see the mesh visualized. I'd agree with you that this feature is not available for VRML. It is available for OGL. But I cannot tell whether it is available for other visualization methods that I have not tried. This is a good question in Geant4 forum. GEARS simply uses what Geant4 provides.

**Comment by Mahmouded** [comment]
Thanks a lot for your help checking my error. Indeed, it is not related to GEARS, but I just wanted to confirm. It's definitely something to be asked on Geant-4 forum.Many thanks.On 15/03/2023, 04:37 Jing Liu ***@***.***> wrote:

   
  I compiled Geant4/11.1.0/share/Geant4/examples/extended/runAndEvent/RE03 and changed the vis driver in run1.mac to VRML2FILE and saw the same warning message. I opened the resulting g4_00.wrl and could not find anything related to the mesh. When I switch back to OGL or TSG, the message disappears and I can see the mesh visualized. I'd agree with you that this feature is not available for VRML. It is available for OGL. But I cannot tell whether it is available for other visualization methods that I have not tried. This is a good question in Geant4 forum. GEARS simply uses what Geant4 provides.
